features/admin.branding.v1/components/branding-page-layout.tsx

```diff
@@ -18,9 +18,15 @@
 
 import { IdentifiableComponentInterface } from "@wso2is/core/models";
 import { DocumentationLink, PageLayout, useDocumentation } from "@wso2is/react-components";
-import React, { FunctionComponent, ReactElement } from "react";
+import { BrandingAIBanner, LoadingScreen } from "features/admin.ai.v1/components";
+import useAIBrandingPreference from "features/admin.ai.v1/hooks/use-ai-branding-preference";
```

Comment:
Let's use relative imports, as absolute imports can cause issues in the build artifacts and affect test execution.

Reply:
this is addressed  https://github.com/wso2/identity-apps/pull/5961/commits/d7d8a5c2caca45996da76831f679060f7e08089b